### ISSUE TITLE
Ansible 2.18: MoultiDisplay.display() got an unexpected keyword argument 'caplevel'

### ISSUE THREAD

**Issue by xavierog** [xavierog/moulti#23]
Moulti+Ansible 2.18 displays a warning instead of the result of the first successful step:
```
[WARNING]: Failure using method (v2_runner_on_ok) in callback plugin
(<ansible.plugins.callback.moulti.CallbackModule object at 0x7f95745ce9f0>):
MoultiDisplay.display() got an unexpected keyword argument 'caplevel'
```

Presumably, adjusting MoultiDisplay.display so it accepts keyword arguments should do (+handling that `caplevel` thing itself, of course).

**Comment by xavierog** [comment]
Fixed in branch `devel`, specifically in commit https://github.com/xavierog/moulti/commit/c5a4db843859b0cc37e76664b2c303168583f9d4

**Comment by xavierog** [comment]
This bugfix is now available as part of [Moulti 1.23.1](https://github.com/xavierog/moulti/releases/tag/v1.23.1).
